totalRP3_Extended_Tools/items/effects.lua

```diff
@@ -699,6 +903,14 @@ function TRP3_API.extended.tools.initItemEffects()
 	document_close_init();
 
 	-- Operands
+	inv_item_name_init();
+	inv_item_icon_init();
+	inv_item_quality_init();
+	inv_item_id_weight_init();
+	inv_item_value_init();
 	inv_item_count_init();
 	inv_item_weight_init();
+    inv_container_slot_id_init();
```

Comment:
I studied this PR long and hard, and all I have to show for it is a comment about inconsistent indentation.